src/View/Helper/SchemaHelper.php

```diff
@@ -32,14 +32,16 @@ class SchemaHelper extends Helper
      * @param array $schema Object schema array.
      * @return array
      */
-    public function controlOptions($name, $value, $schema) : array
+    public function controlOptions($name, $value, $schema = []) : array
     {
         $options = Form::customControlOptions($name);
         if (!empty($options)) {
             $options['value'] = $value;
 
             return $options;
         }
+        $schema = empty($schema) ? [] : $schema;
```

Comment:
Not necessary if we use `array $schema = []` as last method argument (add a type)